### ISSUE TITLE
Wheel packages for Windows and MacOS

### ISSUE THREAD

**Issue by mrakitin** [spglib/spglib#144]
Dear `spglib` developers,

Are you considering building wheel packages for Windows and MacOS? The conda-forge counterpart has it available via https://github.com/conda-forge/spglib-feedstock. That would be very beneficial for users. Thanks!

Thanks,
Maksim Rakitin

**Comment by atztogo** [comment]
Python wrapper for spglib requires compiling C code, and so the packaging is a tricky job. The conda-forge packages are made thanks to @jochym. The wheel for linux was made by @odidev, https://github.com/spglib/spglib/pull/130. For the other OSs,  maybe we can use cibuildwheel, https://github.com/pypa/cibuildwheel, but we will need to write another workflow from scratch, for which I may not have time to do.

**Comment by elichad** [comment]
Voicing a +1 for wheels for these other OSs. On Windows you have to do a ~5GB download of Microsoft Visual C++ Build Tools to be able to build spglib, which is a barrier for some (and an unpleasant surprise when installing packages that use spglib as a dependency).

**Comment by atztogo** [comment]
@elichad, your contribution is welcome. But how about using the conda-forge package?

**Comment by elichad** [comment]
We can and do. But not all of our users will use conda

**Comment by atztogo** [comment]
I attempted making wheel packages for macOS and windows. They may work or may not work. Let's see.

https://pypi.org/project/spglib/#files

I am not at all a specialist on it, and your contribution for the workflow of github action is welcome.

https://github.com/spglib/spglib/blob/develop/.github/workflows/pypi-wheels-win-mac.yml

**Comment by elichad** [comment]
The wheels work on my local Windows machine and all our CI test runners (PR linked above if you want to see what we test). No problem when running our package tests with spglib 2.0.0 wheels as a dependency. Thanks very much!

**Comment by LecrisUT** [comment]
@elichad @mrakitin  Can you test the new `rc` releases, maybe after #226 is merge. There has been a big change in the tools we use. The CIs seem to be passing the tests, but it would be nice to have a confirmation of it.

The release candidates will be available on the [test.pypi repos](https://test.pypi.org/project/spglib).

**Comment by elichad** [comment]
@LecrisUT I'm no longer on that project; tagging @patrick-austin and @joelvdavies so they're aware this change is upcoming

**Comment by patrick-austin** [comment]
I've run our CI with `spglib-2.1.0rc1` and all the tests passed as expected.

**Comment by mrakitin** [comment]
Hi @LecrisUT, All, I've tested the 2.1.0 rc1 version on my Mac computer with Python 3.10, and it worked pretty well - all `pytest`-tests passed. Thanks for the build!

**Comment by LecrisUT** [comment]
Ok, @mrakitin, could you close the issue if it is resolved for now?

**Comment by mrakitin** [comment]
Sure, @LecrisUT! Is there a plan to push the same packages to the main PyPI repository?

**Comment by LecrisUT** [comment]
Well of course it will be, but wait on the main devs. I'm only a contributor that helped with the packaging side over here.
